scrapy_zyte_api/responses.py

```diff
@@ -20,11 +22,27 @@ class ZyteAPIMixin:
     def __init__(self, *args, raw_api_response: Dict = None, **kwargs):
         super().__init__(*args, **kwargs)
         self._raw_api_response = raw_api_response
+        if not hasattr(self, "attributes"):
+            self.attributes: Tuple[str, ...] = (
+                "url",
+                "status",
+                "headers",
+                "body",
+                "request",
+                "flags",
+                "certificate",
+            )
+            if parse_version(scrapy.__version__) >= parse_version("2.1.0"):
```

Comment:
That's quite minor, but could you please move Scrapy version check code to a separate function (e.g. `utils.scrapy_210_or_later`, or `scrapy_supports_attributes_ip_address`)? It would help with code duplication a tiny bit (there are similar checks in tests), and would allow to isolate pkg_resources import, which use is discouraged now (https://setuptools.pypa.io/en/latest/pkg_resources.html). 

As for pkg_resources, it seems the replacement for parse_version is not available in the suggested replacement modules, so that look fine to keep using it.